spec/support/static_validation_helpers.rb

```diff
@@ -12,7 +13,8 @@
 module StaticValidationHelpers
   def errors
     target_schema = schema
-    validator = GraphQL::StaticValidation::Validator.new(schema: target_schema)
+    target_rules = defined?(rules) ? rules : GraphQL::StaticValidation::ALL_RULES
```

Comment:
There have been so many bugs where one validator messed with another, that's why I recently switched these to run against all rules 😞